applications/etherpad-lite/var/README

```diff
@@ -0,0 +1 @@
+This directory is a default path for dirty db.
```

Comment:
You cannot add an empty directory to git.  So without this file, when a new user checks out the benchmark, there will be no `var` directory, etherpad throws an error.  People add a dummy file like this to track an empty directory.

However, I thought about it again, and I think modifying `settings.json` would be a cleaner solution.  I will remove the second commit from this pull request and issue a separate pull request.